README.md

```diff
@@ -285,6 +285,37 @@ Since a full Raspberry Pi 4 support is only available in upstream Linux kernels
 
 Once your Raspberry Pi 4 is happily running an EVE image you can start using EVE controller for further updates (so that you don't ever have to take an SD card out of your board). Build your rootfs by running `make ZARCH=arm64 HV=xen rootfs` (or `make ZARCH=arm64 HV=kvm rootfs` if you want KVM by default) and give resulting `dist/arm64/current/installer/rootfs.img` to the controller.
 
+### How to use on an Onlogic FR201 ARM board
+
+Onlogic Factor 201 (FR201) is a board based on the Raspberry Pi Compute Module 4 (CM4). To use EVE on FR201, build an image for Raspberry Pi 4, as described above and flash it on a USB stick. Then, to enable FR201's specific devices, the boot configuration has to be manually edited.
+
+Specifically, in file `config.txt` of the 1st partition of the live/installer image, the following 2 lines must be changed:
+
+```
+# comment the following line:
+# from:
+    dtoverlay=bcm2711-spi-tpm-slb9670,ce=0x01
+# to:
+    #dtoverlay=bcm2711-spi-tpm-slb9670,ce=0x01
+
+# and uncomment the following line:
+# from:
+    #include fr201.txt
+# to:
+    include fr201.txt
+```
+
+To do so, execute the following. Replace `X` in `sdX1` with the device letter of the USB stick (eg `sda1`):
```

Comment:
I don't think we need to include the details on how to edit a file, specially because the instructions will be different if the user is using another OS, like MacOS..... just instruct the user to edit the config.txt file of the EFI partition should be enough.